evm/hyperliquid-block-helper.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const { Wallet } = require('ethers');
+const { exec } = require('child_process');
+const { promisify } = require('util');
+const { Command, Option } = require('commander');
+const { printInfo, printError, validateParameters, mainProcessor } = require('./utils');
+const { addEvmOptions } = require('./cli-utils');
+const execAsync = promisify(exec);
+
+function addressToBytes(address) {
+    return Buffer.from(address.replace('0x', ''), 'hex');
+}
+
+function actionHash(action, vaultAddress, nonce) {
```

Comment:
```suggestion
function actionHash(action, activePool, nonce) {
```
make the names match across the functions?